core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -1891,6 +1910,29 @@ private static void warnIfServiceHasMultipleRoutes(String path, HttpService serv
         }
     }
 
+    /**
+     * Default success response classification function which checks
+     * {@link RequestLog#responseCause()} is null, 100 &lt;= {@link HttpStatus} &lt; 400
+     * and {@link RpcResponse#isCompletedExceptionally()} == {@code false}.
+     */
+    private static boolean isSuccess(RequestContext ctx, RequestLog log) {
```

Comment:
How about introducing `SuccessFunction` interface so that Spring users easily make an instance of `SuccessFunction` Spring `@Bean`?
The `SuccessFunction` could be placed in `armeria.common`.
```java
public interface SuccessFunction {

   static of() {
       // Move `isSuccess` method to here.
       ...
   }

   boolean isSuccess(RequestContext ctx, RequestLog log);
```

Reply:
Done. Not making `AbstractArmeriaAutoConfiguration` support this yet. Do you think it's better to do it for server level?

Reply:
> Do you think it's better to do it for server level?

I think so. But we don't need to add it to this PR. We can easily add it later when there is a demand for it.